script/deploy/tron/deploy-and-register-allbridge-facet.ts

```
@@ -20,16 +20,13 @@ import {
   displayRegistrationInfo,
   getFacetSelectors,
 } from '../../utils/utils'
+import { proposeDiamondCut } from '../shared/propose-diamond-cut'
 import { getContractVersion } from '../shared/getContractVersion'
```

ESLint

[error] `../shared/getContractVersion` import should occur before import of `../shared/propose-diamond-cut`

(import/order)